dapper-wallet-storefront/public/cadence/metadata_script.cdc

```diff
@@ -51,7 +51,16 @@ pub fun main(
     .get() as! {String: String}
   let name = template["title"]!
   let description = template["description"]!
-  var imageURL = template["mediaUrl"]!
+  var imageURL = ""
+  let posterURL = template["posterUrl"]
+  var mediaURL = template["mediaUrl"]!
```

Comment:
do the `!` assertion on line 61 - this way, in case there is a posterUrl and not a mediaUrl, then it shouldn't stop the script